README.md

```diff
@@ -80,11 +80,12 @@ PRs should include:
   > think?"
 
 - An approval means you're confident that this PR will both do what it's supposed to do, improve the Mozilla Social codebase, and not introduce technical debt.
-- Expect to start your review within one business day of the PR going to review and complete it within another. If you can't do that, please tell the author early on.
-  As an author, expect that meaningful review may take time. If there is so much
-  urgency that the code must be checked in as soon as possible this should be
-  both an exceptional situation and you should be collaborating with your
-  reviewer (and others) during the drafting process.
+- Ideally, a review should be completed within one business day of the request. 
```

Comment:
cc @ml8

Reply:
@efixler works for me

Reply:
LGTM. (Is "by the end of the next business day" an acceptable compromise?)

Reply:
@ml8 let's keep this as is for now, pending further input. I think my biggest hesitation around that suggestion isn't necessarily the raw time delta, it's the fact that we are all in different time zones, so "end of business day" can mean different things.